components/modals/deleteServiceModal.tsx

```diff
@@ -0,0 +1,104 @@
+import IconX from "tabler-icons/tsx/x.tsx";
+import {Audience} from "snitch-protos/protos/sp_common.ts";
+import {Pipeline} from "snitch-protos/protos/sp_pipeline.ts";
+import {opModal} from "../serviceMap/opModalSignal.ts";
+import {opUpdateSignal} from "../../islands/serviceMap.tsx";
+import {Toast, toastSignal} from "../toasts/toast.tsx";
+import {useState} from "preact/hooks";
+
+export const DeleteServiceModal = (
```

Comment:
I see this hasn't been formatted using deno fmt. Can you pretty please 🙏 either add the commit hooks I sent you to do that or add file watchers to do fmt/lint on save in your IDE?